### PR TITLE
adding f-score problem

### PR DESCRIPTION
**Pull request by rafaelgreca**
I decided to change the F1-Score problem (#67) to deal with the [F-Score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.fbeta_score.html) metric, which is a little bit more challenging and broader. Let me know what you think :)

### CODE REVIEW

**Review by Sanket-Kathrotiya — APPROVED** [review]

**Review by moe18 — APPROVED** [review]
Looks good 👍
